src/tox/config/__init__.py

```diff
@@ -309,8 +313,8 @@ def parseconfig(args, plugins=()):
 
 
 def get_py_project_toml(path):
-    with io.open(str(path), mode="rb") as file_handler:
-        config_data = tomllib.load(file_handler)
+    with io.open(str(path), mode=toml_mode, encoding=toml_encoding) as file_handler:
```

Comment:
Do we need io.open here, can't we just use io?

Reply:
Just use io? Do you mean just use open? That's missing some arguments needed on Python 2. IIRC encoding.

Reply:
Exactly encoding.